src/components/Observations/UserCard.js

```diff
@@ -29,10 +29,15 @@ const UserCard = ( ): Node => {
   if ( !user ) { return <View className="flex-row mx-5 items-center" />; }
   const navToUserProfile = ( ) => navigation.navigate( "UserProfile", { userId: user.id } );
 
+  const productionIcon = remoteUser?.icon_url.replace( "staticdev", "static" );
```

Comment:
You need to guard against `icon_url` being blank or null here, i.e. the user has no icon. In that situation, I think you probably just don't want to show an image at all.